react/training/src/redux/Auth/reducer.js

```diff
@@ -0,0 +1,18 @@
+const defaultState = {
+  auth: undefined,
+  loading: false,
+  hasError: false
+};
+
+export function reducer(state = defaultState, action) {
+  switch (action.type) {
+    case 'SIGN_IN_SUCCESS':
+      return { ...state, auth: action.auth };
+    case 'IS_LOADING':
```

Comment:
Name this action `SIGN_IN`. `IS_LOADING` name doesn't represent a sign in loading action